test/spec/collection.test.js

```diff
@@ -6,6 +6,12 @@ import { explainData } from "../../index";
 import { invalid, count, minCount, maxCount } from "../../lib/symbols";
 
 describe("collection", () => {
+  describe("factory", () => {
```

Comment:
Coveralls was complaining about reduced test coverage so I added a new test to increase coverage.